src/scaffoldmaker/meshtypes/meshtype_3d_brainstem.py

```diff
@@ -289,11 +311,13 @@ def generateBaseMesh(region, options):
         :return: None
         """
         parameterSetName = options['Base parameter set']
+        isBrainstem = 'Brainstem 1' in parameterSetName
         isCat = 'Cat 1' in parameterSetName
         isHuman = 'Human 1' in parameterSetName
         isRat = 'Rat 1' in parameterSetName
 
         centralPath = options['Central path']
+        brainstemPath = options['Brainstem path']
```

Comment:
I know why I added 'Brainstem 1' in parameterSetName. Because I tried to used cls.centralPathDefaultScaffoldPackages['Brainstem 1'] before, but cls gave me an error as not defined

Reply:
Change signature of function to:
```
    @classmethod
    def generateBaseMesh(cls, region, options):
```
Shelley was following an old style we used for early scaffolds.
@classmethod tells it the class used is passed as the first argument.